tests/test_routers/test_utterances.py

```diff
@@ -147,3 +148,18 @@ def test_perturbed_utterances(app: FastAPI, monkeypatch):
     ).json()
     # Utterance 1 has 11 perturbation tests
     assert len(resp) == 11
+
+
+def test_post_utterances(app: FastAPI) -> None:
+    client = TestClient(app)
+
+    request = [{"persistent_id": 0, "data_action": "remove"}]
+    resp = client.post("/dataset_splits/eval/utterances", json=request)
+    assert resp.status_code == HTTP_200_OK, resp.text
+    assert resp.json() == [{"persistentId": 0, "dataAction": "remove"}]
+
+    # Reset tag to NO_ACTION
+    request = [{"persistent_id": 0, "data_action": "NO_ACTION"}]
+    resp = client.post("/dataset_splits/eval/utterances", json=request)
+    assert resp.status_code == HTTP_200_OK, resp.text
+    assert resp.json() == [{"persistentId": 0, "dataAction": "NO_ACTION"}]
```

Comment:
Just to be super explicit, at a glance, I'd reuse `request` for asserting the response.
```suggestion
    assert resp.json() == request

    # Reset tag to NO_ACTION
    request = [{"persistent_id": 0, "data_action": "NO_ACTION"}]
    resp = client.post("/dataset_splits/eval/utterances", json=request)
    assert resp.status_code == HTTP_200_OK, resp.text
    assert resp.json() == request
```

Reply:
Right, this doesn't work as easily because of camel case vs snake case. You can check my last commit and see if that makes sense or if there is an easier way.

Reply:
Ah! I didn't notice. Here is what you can do: https://github.com/ServiceNow/azimuth/pull/429#pullrequestreview-1296293067